### PR TITLE
AI projects - repo discovery

### PR DESCRIPTION
**Pull request by Prudhvivuda**
Iterating on the work done here for the AI repositories - https://github.com/oss-aspen/Rappel/tree/main/notebooks/project_discovery/repo_discovery

### CODE REVIEW

File: notebooks/emerging_ai_projects/repo_discovery.ipynb [hunk 1]
```diff
@@ -0,0 +1,14803 @@
+{
```
Comment (cdolfi) [inline notebooks/emerging_ai_projects/repo_discovery.ipynb:1]: <p>(apply to graph below as well) slice the strings after ".com/repos/" and make the graph itself bigger </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oss-aspen/Rappel/pull/258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

**Review by dominikkawka — APPROVED** [review]

**Review by hemajv — CHANGES_REQUESTED** [review]
@Prudhvivuda nice work! I've left some comments, ptal

**Comment by JamesKunstle** [comment]
I posted comments but ReviewNB didn't save them.

The core problem I have is that the Event Stream isn't representative of the population of GitHub (perhaps it really never was in the original work) so we can't draw conclusions about the results. I think the database that this work is using is based on a set of repos that are assumed to be AI facing, rather than a roughly random set of repos.

I don't have a solution for this without rebuilding the database, but I think mentioning this limitation is imperitive.

**Comment by cdolfi** [comment]
<p>Change the Legend to say "Seed Repo" "Non seed Repo" instead of true/false</p>

---

 <a href='https://app.reviewnb.com/oss-aspen/Rappel/pull/258/discussion/#comment-2339323629'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by Prudhvivuda** [comment]
<p>done</p>

---

 <a href='https://app.reviewnb.com/oss-aspen/Rappel/pull/258/discussion/#comment-2339323629'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by Prudhvivuda** [comment]
<p>I think this shows the number of events from the general population and sub population (known contributors).</p>

---

 <a href='https://app.reviewnb.com/oss-aspen/Rappel/pull/258/discussion/#comment-2379962858'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by cdolfi — APPROVED** [review]
lgtm

**Pull request merged**